HACKING.md

```diff
@@ -5,47 +5,53 @@
 
 ### Run the HyperShift Operator in a local process
 
-1. Ensure the `KUBECONFIG` evnvironment variable points to a management cluster
+1. Ensure the `KUBECONFIG` environment variable points to a management cluster
    with no HyperShift installed yet.
 
 2. Build HyperShift.
 
-        make build
+        $ make build
 
-3. Install HyperShift with the operator deployment scaled to zero so that it
-   doesn't conflict with your local operator process. 
+3. Install HyperShift in development mode which causes the operator deployment
+   to be deployment scaled to zero so that it doesn't conflict with your local
+   operator process. 
 
-        make install PROFILE=development
+        $ bin/hypershift install --development
 
-4. Run the HyperShift operator locally. Replace `IMAGE` with a custom image,
-   if desired. This image is used for the Control Plane Operator.
+4. Run the HyperShift operator locally.
 
-        make run-local IMAGE=registry.ci.openshift.org/hypershift/hypershift
+        $ bin/hypershift-operator run
 
-### Run a custom image using the production profile
+### Install HyperShift with a custom image
```

Comment:
It'd be good to also add instructions to uninstall, given that we no longer have an `uninstall` make target:
```
bin/hypershift install --render | oc delete -f -
```

Reply:
Uninstall is already documented in the root level README

Reply:
👍